backend/src/api/player_view_functions.clj

```diff
@@ -6,18 +6,27 @@
   "Returns cards as seen by a player"
   [game-state player-id]
   (vec (map
```

Comment:
This function is becoming very long and hard to read... do we really need nested ifs?
I feel like `cond` would be clearer, even if the conditions become a little more verbose, and maybe even split off the different cases into functions?

Reply:
Let's see if it's easier to read now